draft-yusef-lamps-rfc7030-renewal-recommendation.mkd

```diff
@@ -203,7 +203,83 @@ Not sure what yet.
 
 # IANA Considerations
 
-Might need a header allocation
+## Updates to EST .well-known registration
+
+To the Well-Known URIs Registry, at:
+"https://www.iana.org/assignments/well-known-uris/well-known-uris.xhtml", this document updates the well-known name "est".
+
+IANA is asked to change the registration of "est" to include RFC7030, RFC9148,
+and this document.
+
+## EST .well-known sub-registry
+
+IANA is requested to create a new Sub-Registry inside of the well-
+known URIs Registry entitled: "EST well-known URIs".
+
+The registry shall have at least five columns:
+
+1. URI Path Component
+2. short-URI Component
+3. CoAP Path abbreviation
+4. description
+5. reference.
+
+The Short-URI components are from {{RFC9148}}.
+The CoAP Path abbreviations are from {{I-D.ietf-core-uri-path-abbrev}}, and are allocated according to the IANA Considerations in that document, which is Specification Required.
```

Comment:
```suggestion
The CoAP Path abbreviations are from {{I-D.ietf-core-uri-path-abbrev}} and maintained in the Uri-Path-Abbrev registry (in the CoRE parameters group), and only listed here for reference.
```

These need to live in Uri-Path-Abbrev because it ensures uniqueness there. In terms of databases, we can foreign-key into there, but I'm not sure how happy IANA is with that, so this wording demotes them to a hint. (Alternatively, they could be removed from this list altogether, with a hint to look it up over there).